environment.yml

```diff
@@ -3,3 +3,4 @@ channels:
   - conda-forge
 dependencies:
   - configargparse
+  - importlib-metadata  # [py<38]
```

Comment:
@awvwgk do you know if this is the right way to specify Python dependencies on package in `environment.yml`

Reply:
These backport packages are annoying, I can check if there are stubs, but they used to break the possibility to package as noarch: python.

Reply:
Looks like importlib-metadata has proper stubs you can just depend on it for all Python versions.

Reply:
Interesting, I remember I had to stop being `noarch` and do this on the fortls conda-forge recipe to get it to work:

```yml
requirements:
  build:
    - python                                 # [build_platform != target_platform]
    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
  host:
    - pip
    - python
    - setuptools_scm
    - setuptools_scm_git_archive
  run:
    - importlib-metadata  # [py<38]
    - python
    - typing-extensions  # [py<38]
 ```